src/cascadia/TerminalSettingsEditor/GlobalAppearance.cpp

```diff
@@ -28,4 +37,139 @@ namespace winrt::Microsoft::Terminal::Settings::Editor::implementation
     {
         _State = e.Parameter().as<Editor::GlobalAppearancePageNavigationState>();
     }
+
+    winrt::hstring GlobalAppearance::LanguageDisplayConverter(const winrt::hstring& tag)
+    {
+        if (tag == systemLanguageTag)
+        {
+            return RS_(L"Globals_LanguageDefault");
+        }
+
+        winrt::Windows::Globalization::Language language(tag);

```

Comment:
nit: use universal construction syntax (stylistic concern to avoid 'most vexing parse')

```suggestion
        winrt::Windows::Globalization::Language language{ tag };
```